### DISCUSSION TITLE
Would the library eventually change its license?

### DISCUSSION THREAD

**Discussion by Raj2032** [slint-ui/slint#1282] [Q&A]
Hi I am curious to know if the library would eventually change hte license? It looks very great and I wouldn't mind donating to the project (provided that I actually end up making something) but I am not at all a huge fan of the license at all which is a huge turn off for me. I wouldn't mind if it was LGPL but yeah just asking if you guys were planning to change its license eventually?

**Comment by ogoffart** [comment]
Regarding the license, we are open for suggestions, but it is our intention to keep this current model.

1. GPL for open-source users who also open source their code.
2. With the possibility to do commercial programs using the other license options.
 
So right now, we hope the Ambassador license should be fitting the needs of people wanting to use a license like the LGPL to write commercial product.  Using this option, all you have to do is letting us know you are using Slint, and allowing us to communicate about it, and you can use Slint for free.

Is there a reason you would be using the LGPL but wouldn't want to use the Ambassador license?
